packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/EditView/index.js

```diff
@@ -113,7 +114,7 @@ const ApiTokenCreateView = () => {
     async () => {
       const {
         data: { data },
-      } = await axiosInstance.get(`/admin/api-tokens/${id}`);
+      } = await fetchClient.get(`/admin/api-tokens/${id}`);
```

Comment:
I'm wondering if that would be a good opportunity to move the `/admin/` prefix into `fetchClient`, because it is always required and it would be quite beneficial if we could make this configurable in one place.

Reply:
do you mean directly inside the baseURL? because I saw inside the code different places where the url passed as argument is a variable, maybe it is better to leave it like this, just because in the future could be beneficial if we want to  change something

Reply:
For example:
-  packages/core/admin/admin/src/content-manager/components/CollectionTypeFormWrapper/index.js line 140
`const { data } = await axiosInstance.get(requestURL, { cancelToken: source.token });` and requestUrl is coming from
`  const requestURL = useMemo(() => {
    if (isCreatingEntry && !origin) {
      return null;
    }

    return getRequestUrl(`collection-types/${slug}/${origin || id}`);
  }, [slug, id, isCreatingEntry, origin]);`

Reply:
we have found the getRequestUrl which has a lot of different implementations inside the repo that gave us a similar logic (`packages/core/admin/admin/src/utils/getRequestUrl.js` this one is the correct one for us).
I think it can be useful to open a ticket on it and redefine every single implementation or getRequestUrl with a new one more specific (for example getAdminUrl), what do you think?

Reply:
I agree with you - let's do this as a second step and clarify first, how we'd like to build URLs in the future :)